Description: fix qpygui_qpair.sip building against Qt older than 5.2
Origin: upstream, changeset 5c798e751276
Last-Update: 2013-01-09

--- a/sip/QtGui/qpygui_qpair.sip
+++ b/sip/QtGui/qpygui_qpair.sip
@@ -19,6 +19,8 @@
 // WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
 
 
+%If (Qt_5_2_0 -)
+
 %MappedType QPair<QOpenGLTexture::Filter, QOpenGLTexture::Filter>
         /DocType="tuple-of-QOpenGLTexture.Filter-QOpenGLTexture.Filter"/
 {
@@ -110,6 +112,10 @@
 %End
 };
 
+%End
+
+
+%If (Qt_5_2_0 -)
 
 %MappedType QPair<float, float> /DocType="tuple-of-float-float"/
 {
@@ -202,3 +208,5 @@
     return sipGetState(sipTransferObj);
 %End
 };
+
+%End
